packages/target-nodejs-sdk/src/target.js

```diff
@@ -45,6 +47,8 @@ export function executeDelivery(options, decisioningEngine) {
     useBeacon,
     createDeliveryApiMethod = createDeliveryApi
   } = options;
+  const timingTool = createPerfToolInstance();
```

Comment:
Let's create timingTool outside this fn.  It should be a singleton, so we don't need to set it up for every executeDelivery call

Reply:
I placed it at the top of the file!